lms/static/sass/views/_teams.scss

```diff
@@ -346,6 +346,106 @@
     }
   }
 
+  .team-profile {
+    .discussion-module {
+      width: 60%;
+      @include float(left);
+    }
+    .team-detail {
+      width: 30%;
+      @include float(left);
+      @include margin($baseline, 0, $baseline, $baseline + $baseline/2);
+
+      div, h4 {
+        margin-bottom: ($baseline/4);
+      }
+      @extend %t-copy-sub1;
+      color: $gray-l1;
+
+      .image-url {
+        border: 2px solid $outer-border-color;
+        border-radius: ($baseline/4);
+        margin-bottom: ($baseline/4);
+      }
+
+      .leave-team, .invites-header, .invite-text {
+        margin-bottom: ($baseline/2);
+      }
+
+      .invite-team {
+        margin-top: ($baseline/2);
+        margin-bottom: ($baseline/2);
+      }
+      .invite-link-input {
+        width: 100%;
+      }
+
+      .member-profile {
+        position: relative;
+        display: inline-block;
+
+        p {
```

Comment:
@marcotuts We are using the same tooltip in sequence nav bar in courseware [here](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/css/sequence/display.scss#L189) 
Maybe we can put this sass to some common place to re use but i have no idea what will be the impact as this is in xmodule. Any suggestions ?